presto-docs/src/main/sphinx/develop/event-listener.rst

```diff
@@ -46,3 +46,13 @@ Example configuration file:
     event-listener.name=custom-event-listener
     custom-property1=custom-value1
     custom-property2=custom-value2
+
+Multiple Event Listeners
+------------------------
+
+Multiple instances of the same, or different event listeners can be
+installed and configured by setting ``event-listener.instance.names``
```

Comment:
We should do this like the recent access control change, where each file name is listed. Unlike catalogs, there’s no concept of “instance name” used by Presto, so we can reference the files directly. See https://github.com/prestosql/presto/blob/master/presto-main/src/main/java/io/prestosql/security/AccessControlConfig.java and https://github.com/prestosql/presto/blob/master/presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

Reply:
I updated the doc here to reflect the new conf changes